### ISSUE TITLE
Autoprefixer munges some CSS class names

### ISSUE THREAD

**Issue by canterberry** [parcel-bundler/parcel#2072]
# 🐛 bug report

TL;DR. If I try to use a class name of `columns` in one of the CSS files included in my project, the resulting CSS module exports the class name incorrectly with vendor prefixes. The autoprefixer should only affect CSS *properties*, not *class name selectors*.

I would imagine that other auto-prefixed CSS properties would trigger this behavior as well, although I have not tried others.

## 🎛 Configuration (.babelrc, package.json, cli command)

**.babelrc**
```json
{
  "presets": ["env", "react", "stage-2"]
}
```

**.postcssrc**
```json
{
  "modules": true,
  "plugins": {
    "autoprefixer": true
  }
}
```

## 🤔 Expected Behavior

This is what the output for my CSS module after a `parcel build ...` should look like:

```javascript
> module.exports={columns:"_columns_mpr9j_1"};
```

## 😯 Current Behavior

This is the what the resulting CSS module after a `parcel build` command looks like now:

```javascript
module.exports={"-webkit-columns":"_columns_1m6to_1","column-count":"_columns_1m6to_1"};
```

Note that this problem **does not occur** when running a development server (via `parcel start ...`).

## 💁 Possible Solution

Probably an autoprefixer issue. Not familiar with the implementation, so no suggestions there.

## 🔦 Context

I have a React component called `<Columns>`, and by convention, the root CSS class name on my project's React components is the name of the component.

This is pretty easy to work around simply by renaming the CSS class, so not a critical issue. Just a minor annoyance that was difficult to troubleshoot at first because it doesn't happen with `parcel start`.

## 💻 Code Sample

[canterberry/parcel-autoprefixer-issue](https://github.com/canterberry/parcel-autoprefixer-issue) contains a minimal configuration for demonstrating this issue.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.10.0 |
| Node             | 10.9.0 |
| Yarn         | 1.10.0 |
| Operating System | Linux |

**Comment by DeMoorJasper** [comment]
This is probably an autoprefixer bug

**Comment by canterberry** [comment]
You may be right. I'll file a complementary bug in the autoprefixer project as well, just to have some eyes in that project, too.

**Comment by DeMoorJasper** [comment]
https://github.com/css-modules/postcss-modules ?

**Comment by canterberry** [comment]
Created [css-modules/postcss-modules #87](https://github.com/css-modules/postcss-modules/issues/87) to get the eyes of that package as well. Thanks for the direction, @DeMoorJasper.

**Comment by canterberry** [comment]
Closing this issue, as there hasn't been any response here (or in css-modules/postcss-modules#87) in six months, and I've since switched to using *styled-components* instead of PostCSS modules.
